Bottom: 2e365fa98d52b5b26d7d1fcb913842a4c6f293b6
Top:    3e284195875c201602d05798a8c7b6672a5a4fd0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 13:38:21 +0100

Refresh of off-to-find-burried-treasure-o

---

diff --git a/src/pour.c b/src/pour.c
index 176debd..9108506 100644
--- a/src/pour.c
+++ b/src/pour.c
@@ -1342,7 +1342,7 @@ RestoreToPours (DataType * Data, int type, void *ptr1, void *ptr2)
 //      printf ("Calling InitPourClip from RestoreToPour\n");
       InitPourClip (PCB->Data, (LayerTypePtr) ptr1, (PourTypePtr) ptr2);
     }
-  PlowPours (Data, type, ptr1, ptr2, add_plow, False);
+  PlowPours (Data, type, ptr1, ptr2, add_plow, false);
 }
 
 void
@@ -1356,7 +1356,7 @@ ClearFromPours (DataType * Data, int type, void *ptr1, void *ptr2)
 //      printf ("Calling InitPourClip from ClearFromPour\n");
       InitPourClip (PCB->Data, (LayerTypePtr) ptr1, (PourTypePtr) ptr2);
     }
-  PlowPours (Data, type, ptr1, ptr2, subtract_plow, False);
+  PlowPours (Data, type, ptr1, ptr2, subtract_plow, false);
 }
 
 #warning FIXME Later: We could perhaps reduce un-necessary computation by using this function
@@ -1365,5 +1365,5 @@ MarkPourIslands (DataType * Data, int type, void *ptr1, void *ptr2)
 {
   if (!Data->ClipPours)
     return;
-  PlowPours (Data, type, ptr1, ptr2, mark_islands, True);
+  PlowPours (Data, type, ptr1, ptr2, mark_islands, true);
 }
